WalletWasabi.Fluent/ViewModels/CoinJoinProfiles/CoinJoinProfilesViewModel.cs

```diff
@@ -46,6 +46,8 @@ public static CoinJoinProfileViewModelBase IdentifySelectedProfile(KeyManager ke
 		var currentProfile = new ManualCoinJoinProfileViewModel(keyManager);
 		var result = DefaultProfiles.FirstOrDefault(x => x == currentProfile) ?? currentProfile;
 
+		result.AnonScoreTarget = currentProfile.AnonScoreTarget;    // Edge case: Update the PrivateCJProfile anonscore target, otherwise the randomly selected value will be displayed all time.
```

Comment:
```suggestion
		// Edge case: Update the PrivateCJProfile anonscore target, otherwise the randomly selected value will be displayed all time.
		result.AnonScoreTarget = currentProfile.AnonScoreTarget;
```